release-notes/v3.1.0.md

```diff
@@ -0,0 +1,80 @@
+# HPE Volume Driver for Kubernetes FlexVolume Plugin 3.1.0 Release Notes
+
+## HPE Volume Driver for Kubernetes FlexVolume Plugin 3.1.0
+
+| **Version:** |3.1.0|
+|--------------|-----|
+| **Revision:** | Friday Nov 1, 2019 22:05:17|
+
+The release notes describe the major changes, fixes, and known issues for this release of the new for HPE Volume Driver for Kubernetes FlexVolume Plugin 3.1.0. They do not include all individual fixes and internal changes.
```

Comment:
rephrase:
for this release of the new for HPE...

Reply:
done..